www/front_src/src/Resources/Graph/Performance/Legend/index.tsx

```diff
@@ -68,8 +89,21 @@ const LegendContent = ({
 }: LegendContentProps): JSX.Element => {
   const classes = useStyles({ panelWidth });
   const theme = useTheme();
+  const { metricsValue, getFormattedMetricData } = useMetricsValueContext();
 
-  const getLegendName = ({ metric, name, display }: Line): JSX.Element => {
+  const getFormattedName = ({ name, unit }) => {
```

Comment:
This function is already implemented via `truncateInMiddle`

Reply:
It does not work with multiline clamping